recipes/wayland/all/conanfile.py

```diff
@@ -55,48 +53,61 @@ def requirements(self):
         self.requires("expat/2.4.8")
 
     def build_requirements(self):
-        self.build_requires("meson/0.60.2")
+        self.tool_requires("meson/0.63.0")
+        self.tool_requires("pkgconf/1.7.4")
+        if cross_building(self):
+            self.tool_requires(f"wayland/{self.version}")
```

Comment:
What happens if someone adds a new Wayland package version and the first build attempt is a cross-compile? 

Probably Conan complaining that the package isn't existing? CI could also fail if cross-compiling for macOS M1 would be one of the first jobs running. If I would be a consumer of the package and no matching native package would exist, I would probably also be pretty confused what to do after getting such an error message.

Reply:
Good question. Seeing your question, I assumed that without `--build missing` or something similar, it would fail.... but with Conan 1.50.0 I'm not getting that behavior when I remove the package locally and run `conan create` for a cross-compile without any `-b/--build` flags. Instead, I'm seeing it build the package natively before building the cross-compiled package.

So, maybe this isn't actually a problem? I'm surprised. If you - or anyone reading this - has a cross-compilation setup for Conan, I'd appreciate a second-opinion to confirm this behavior.